saas_app/locale/fr/LC_MESSAGES/django.po

```diff
@@ -438,39 +457,48 @@ msgstr ""
 msgid "Something went wrong and there are no logs to display!"
 msgstr "Quelque chose s'est mal passé et il n'y a aucun journal à afficher !"
 
-#: saas_app/views.py:27
+#: saas_app/views.py:30
 msgid "Something went wrong with the search. Please try again."
 msgstr "Une erreur s'est produite lors de la recherche. Veuillez réessayer."
 
-#: saas_app/views.py:80
+#: saas_app/views.py:83
 msgid "You have successfully restored all roles"
 msgstr "Vous avez restauré avec succès tous les rôles"
 
-#: saas_app/views.py:86
+#: saas_app/views.py:89
 #, python-format
 msgid "You have successfully switched to %(role)s role"
 msgstr "Vous avez réussi à passer au rôle %(role)s"
 
-#: saas_app/views.py:92
+#: saas_app/views.py:95
 msgid "Something went wrong with switching roles. Please try again."
 msgstr ""
 "Quelque chose s'est mal passé avec le changement de rôle. Veuillez réessayer."
 
-#: submit_request/forms.py:42
+#: submit_request/forms.py:48
 msgid ""
 "If Saas is to be used for all of CDS, select FC 110804. If the Saas is to be "
 "used specifically for your BU, please select the appropriate FC."
 msgstr ""
+"Si Saas doit être utilisé pour tous les CDS, sélectionnez FC 110804. Si le "
```

Comment:
I think we could update the syntax with the following:
`"Si ce SaaS doit être utilisé par tous le personnel du SNC, sélectionnez FC 110804. Si le "`

Reply:
Done! Thank you for the suggestion.